source/common/http/async_client_impl.h

```diff
@@ -276,6 +276,7 @@ class AsyncStreamImpl : public virtual AsyncClient::Stream,
   Tracing::NullSpan active_span_;
   const Tracing::Config& tracing_config_;
   std::shared_ptr<NullRouteImpl> route_;
+  std::unique_ptr<Router::RetryPolicy> retry_policy_;
```

Comment:
```suggestion
  const std::unique_ptr<const Router::RetryPolicy> retry_policy_;
```

See above about moving initialization into the constructor initializer list.

Reply:
done